pkgs/dartpad_ui/web/frame.js

```diff
@@ -22,16 +22,22 @@ function messageHandler(e) {
   var obj = e.data;
 
   if (obj.command === 'execute') {
-    // TODO: Switch to using engineInitializer.initializeEngine(config). See
-    // https://docs.flutter.dev/development/platform-integration/web/initialization.
-    window.flutterConfiguration = {
-      canvasKitBaseUrl: obj.canvasKitBaseUrl
-    };
-
-    replaceJavaScript(obj.js);
+    runFlutterApp(obj.js);
   }
 };
 
+function runFlutterApp(compiledScript) {
+  var blob = new Blob([compiledScript], {type: 'text/javascript'});
+  var url = URL.createObjectURL(blob);
+  _flutter.loader.loadEntrypoint({
+    entrypointUrl: url,
+    onEntrypointLoaded: async function(engineInitializer) {
```

Comment:
I *think* you can even drop this bit. This is wiring up the default behavior I believe.

Right @ditman  ?